index.js

```diff
@@ -0,0 +1,17 @@
+function fibonacci(number) {
+  let arr = []
+  let n1 = 0,
+    n2 = 1,
```

Comment:
let n2

Reply:
Upon adding comma, let doesn’t have to be present. Prettier is making indentation weird.